### DISCUSSION TITLE
Audio not playing via bluetooth connected devices

### DISCUSSION THREAD

**Discussion by chitraarasu** [bluefireteam/audioplayers#1412] [Q&A]
Audio is coming from the iPhone earpiece and not from the bluetooth speaker. Even after connecting to Bluetooth, the sound comes only from the phone earpiece. I Checked on multiple iPhones and The issue is the same. It is working fine on Android. What should i have to do to fix this. I am facing this issue only on iPhone

**Comment by Gustl22** [comment]
See #1194 and #1408. This will be fixed in the next release, or you provide your own `AudioContext`
